styleguide/screens/Patterns/Tether/Tether.js

```diff
@@ -0,0 +1,182 @@
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: cx(m.par, css.specimenContainer),
+            } }
+            code={ dedent`
+              <Tether
+                target={ <button>Anchor</button> }
+                verticalAttachment={ ${verticalAttachment} }
+                horizontalAttachment={ ${horizontalAttachment} }
+                active
+              >
+                <div />
+              </Tether>
+            ` }
+          >
+            <Tether
+              target={ <Target /> }
+              verticalAttachment={ verticalAttachment }
+              horizontalAttachment={ horizontalAttachment }
+              flushVertical={ flushVertical }
+              flushHorizontal={ flushHorizontal }
+              active
+            >
+              <TetherChild className={ css.tether } />
+            </Tether>
+            <div className={ css.controls } >
+              <div>
+                <label htmlFor="verticalAttachment">verticalAttachment</label>
+                <select
+                  name="verticalAttachment"
+                  value={ verticalAttachment }
+                  onChange={ this.handleVerticalAttachmentChange }
+                >
+                  <option value="TOP">TOP</option>
+                  <option value="BOTTOM">BOTTOM</option>
+                  <option value="CENTER">CENTER</option>
+                </select>
+              </div>
+              <div>
+                <label htmlFor="horizontalAttachment">horizontalAttachment</label>
+                <select
+                  name="horizontalAttachment"
+                  value={ horizontalAttachment }
+                  onChange={ this.handleHorizontallAttachmentChange }
+                >
+                  <option value="CENTER">CENTER</option>
+                  <option value="LEFT">LEFT</option>
+                  <option value="RIGHT">RIGHT</option>
+                </select>
+              </div>
+              <div>
+                <label htmlFor="flushVertical">flushVertical</label>
+                <input
+                  type="checkbox"
+                  name="flushVertical"
+                  checked={ flushVertical }
+                  onChange={ this.handleFlushVerticalChange }
+                />
+              </div>
+              <div>
+                <label htmlFor="flushHorizontal">flushHorizontal</label>
+                <input
+                  type="checkbox"
+                  name="flushHorizontal"
+                  checked={ flushHorizontal }
+                  onChange={ this.handleFlushHorizontalChange }
+                />
+              </div>
+            </div>
+          </Specimen>
+          <T elm="p" className={ m.mtr }>
+            To position the <C>Tether</C> element provide the <C>verticalAttachment</C> and
+            <C>horizontalAttachment</C> props to the <C>Tether</C> component.
+          </T>
+          <T elm="p" className={ m.mtr }>
```

Comment:
This bit is really hard to read with so many code snippets. Thinking that perhaps we should reevaluate how we list out the possible values for each attachment

Reply:
Agreed. Do you think we still need to list them out now that we have an interactive example?

Reply:
Probably not 👍